### DISCUSSION TITLE
在FRI batching时，采用多个随机值r1,r2,...,rn 折叠多个多项式g1,g2,g3,...gn, 和采用一个随机值r, 对其取幂，然后使用幂来折叠多个多项式，在安全性上是是否相同？

### DISCUSSION THREAD

**Discussion by readygo67** [coset-io/zkp-academy#46] [Q&A] [answered]
https://www.youtube.com/watch?v=wqRuoyH3Mqk 视频中提到了两种方法在安全性上有不同，但是没有理解为什么。
![image](https://github.com/user-attachments/assets/9e8919fe-7e03-492e-a51c-64edfdd0763e)

**Comment by wenjin1997** [comment, marked as answer]
在论文[Proximity Gaps for Reed–Solomon Codes](https://eprint.iacr.org/2020/654.pdf) 中提到了关于这两种方法的 Correlated agreement 结论，其结论是不同的，而FRI协议的安全性的证明过程是需要用到对应的Correlated agreement 定理的，因此这两种方法的安全性也是不同的。
![image](https://github.com/user-attachments/assets/12957629-2c32-447e-aa0f-8291ce1f683f)
* 上述定理 1.5 对应问题中的方法二，即 Parametric batching
* 定理 1.6 对应问题中的方法一，即 Affine batching

**Remark**: 这里给出的定理 1.5 与 定理 1.6 是简单版本，论文[Proximity Gaps for Reed–Solomon Codes](https://eprint.iacr.org/2020/654.pdf) 中关于FRI协议安全性证明(见 8.2 节)中实质会用到一个更加复杂的版本，见论文Theroem 7.1 与 Theorem 7.3。
